auth.ts

```diff
@@ -1,9 +1,7 @@
-import { NextAuthOptions } from 'next-auth';
+import NextAuth from 'next-auth';
 import CredentialsProvider from 'next-auth/providers/credentials';
 
-// In a more fleshed out solution we would add the env variables we need (JWT and JWT Session, NEXTAUTH URL, etc )
-export const authOptions: NextAuthOptions = {
-  secret: process.env.NEXTAUTH_SECRET,
```

Comment:
I don't think we need this anymore - but we'll see if it causes problems during deploy. 

The docs cite inferred env variables as one of the [main changes](https://authjs.dev/getting-started/migrating-to-v5)